### ISSUE TITLE
react.useState is not a function

### ISSUE THREAD

**Issue by ryanking1809** [alloc/wana#32]
Hey Alec, just testing out a few things in wana. Looks like there's an issue with `withAuto` in versions 0.13.9+

https://codesandbox.io/s/wana-deferred-update-b7f0t?file=/src/App.js

```
(0 , _react.useState) is not a function - useForceUpdate
```

**Comment by aleclarson** [comment]
Looks like a bug with CSB's bundler.

You can do this as a workaround:
```ts
import { auto, no, o, withAuto } from "wana/dist/wana.js";
```

**Comment by ryanking1809** [comment]
Thanks! I've been bumping into blocking issues having 100s of components update at once, hopefully this can circumvent that.

**Comment by aleclarson** [comment]
Hmm, there's a built-in limiter in wana's batching module 👇 
https://github.com/alloc/wana/blob/df178deb8789aea9e881567e694f05cacc6dbe3a/src/batch.ts#L69

I wonder if making that adjustable would help you. Could you try editing it in your node_modules?

**Comment by ryanking1809** [comment]
Thanks, I'll try playing around with it later today - but maybe a deferred update hook may work better as there are high priority updates for user input and low priority ones for adjacent graphics etc.

**Comment by ryanking1809** [comment]
> Hmm, there's a built-in limiter in wana's batching module 👇
> https://github.com/alloc/wana/blob/df178deb8789aea9e881567e694f05cacc6dbe3a/src/batch.ts#L69
> 
> I wonder if making that adjustable would help you. Could you try editing it in your node_modules?

Surprisingly this seemed to have no effect. I tried some really low numbers like `3`, `50`, `100` (I just did a find & replace in the node_modules to get all the instances) and it still seems to block during mass updates. I'm happy with using `requestIdleCallback` regardless as it gives me finer control over the queue.

I think the issue might be the browser painting more than wana's computing.
